wagtail/locales/views.py

```diff
@@ -18,14 +25,29 @@ def get_value(self, locale):
         return locale
 
 
-class UsageColumn(Column):
+def _can_add_locale() -> bool:
+    defined_locales = [
+        language[0]
+        # WAGTAIL_CONTENT_LANGUAGES is a list of tuples (language_code, language)
+        for language in getattr(settings, "WAGTAIL_CONTENT_LANGUAGES", [])
+    ]
```

Comment:
I think we can use `get_content_languages().keys()` no?

Reply:
oh yeah! I completely forgot about `get_content_languages()` 🤦🏼